ui/apps/dev-server-ui/src/components/App/AppActions.tsx

```diff
@@ -1,4 +1,5 @@
 import { useState } from 'react';
+import { useRouter } from 'next/navigation';
```

Comment:
The additions in this file are not used. However, the "View Details" link in ui/apps/dev-server-ui/src/components/App/AppCardContent.tsx should actually be a button in this file instead